packages/compass-connection-import-export/src/components/file-input.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useMemo } from 'react';
+import {
+  createElectronFileInputBackend,
+  FileInput as CompassFileInput,
+} from '@mongodb-js/compass-components';
+
+type FileInputProps = {
+  label: string;
+  value?: string | null;
+  mode: 'open' | 'save';
+  disabled: boolean;
+  onChange: (filename: string) => void;
+};
+
+export function FileInput({
+  label,
+  value,
+  mode,
+  disabled,
+  onChange,
+}: FileInputProps): React.ReactElement {
+  const onChangeFiles = useCallback(
+    (files: string[]) => {
+      if (files.length > 0) onChange(files[0]);
+    },
+    [onChange]
+  );
+
+  const backend = useMemo(() => {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/consistent-type-imports
+    const electron: typeof import('@electron/remote') = require('@electron/remote');
+    return createElectronFileInputBackend(electron, mode, {
+      title: 'Select connections file',
+      defaultPath: 'compass-connections.json',
+      buttonLabel: 'Select',
+    });
+  }, [mode]);
+
+  return (
+    <CompassFileInput
+      disabled={disabled}
+      label={label}
+      onChange={onChangeFiles}
+      id="conn-import-export-file-input"
+      accept=".json"
```

Comment:
Not sure how this is working internally / what's causing this, but seems like I am not able to select any file on import: 

<img width="438" alt="image" src="https://user-images.githubusercontent.com/5036933/200595571-50f59dab-0cf1-4dc5-a877-7d4adfc000f9.png">

Reply:
Aha, good to know! Kinda wish we could just use browser api here, the interface there seems a bit more straightforward to me

Reply:
I mean, yeah, it’s just that there is no browser API that covers what we need here :confused: